Makefile

```diff
@@ -3,3 +3,4 @@ SEVERITIES = HIGH,CRITICAL
 .PHONY: all
 all:
 	sudo docker build --no-cache --build-arg -t rancher/istio-installer:$(TAG) .
+	sudo docker build --no-cache --build-arg -t rancher/istio-installer-release-mirror:$(TAG) .
```

Comment:
Is the second image still necessary for this approach?

Reply:
I pulled out the drone stuff, but missed this - good catch! it has been removed now.